platform/jobs/RetirementJobs.groovy

```diff
@@ -330,3 +330,109 @@ job('user-retirement-collector') {
+        extraMembersCanView.each { emp ->
+            permission('hudson.model.Item.Read', emp)
+            permission('hudson.model.Item.Discover', emp)
+        }
+    }
+
+    // This label is configured to disallow more than one simultaneous job
+    // per instance, and generally there are always available jenkins-worker
+    // instances idling.
+    label('jenkins-worker')
+
+    // Only one of these jobs should be running at a time per environment
+    concurrentBuild(true)
+    configure { project ->
+        project / 'properties' / 'hudson.plugins.throttleconcurrents.ThrottleJobProperty' <<
+            'paramsToUseForLimit'('ENVIRONMENT')
+        project / 'properties' / 'hudson.plugins.throttleconcurrents.ThrottleJobProperty' <<
+            'limitOneJobWithMatchingParams'('true')
+    }
+
+    // keep jobs around for 30 days
+    logRotator JENKINS_PUBLIC_LOG_ROTATOR(30)
+
+    wrappers {
+        buildUserVars() /* gives us access to BUILD_USER_ID, among other things */
+        buildName('#${BUILD_NUMBER}')
+        timestamps()
+        colorizeOutput('xterm')
+        credentialsBinding {
+            file('USER_RETIREMENT_SECURE_DEFAULT', 'user-retirement-secure-default.yml')
+        }
+    }
+
+    parameters {
+        stringParam('TUBULAR_BRANCH', 'master', 'Repo branch for the tubular scripts.')
+        stringParam('ENVIRONMENT', 'secure-default', 'edx environment which contains the user in question, in ENVIRONMENT-DEPLOYMENT format.')
+    }
+
+    // retry cloning repositories
+    checkoutRetryCount(5)
+
+    multiscm {
+        git {
+            remote {
+                url('git@github.com:edx-ops/user-retirement-secure.git')
+                credentials('jenkins-worker')
+            }
+            branch('master')
+            extensions {
+                relativeTargetDirectory('user-retirement-secure')
+                cloneOptions {
+                    shallow()
+                    timeout(10)
+                }
+                cleanBeforeCheckout()
+            }
+        }
+        git {
+            remote {
+                url('https://github.com/edx/tubular.git')
+            }
+            branch('$TUBULAR_BRANCH')
+            extensions {
+                relativeTargetDirectory('tubular')
+                cloneOptions {
+                    shallow()
+                    timeout(10)
+                }
+                cleanBeforeCheckout()
+            }
+        }
+    }
+
+    environmentVariables {
+        env('PARTNER_REPORTS_DIR', '${WORKSPACE}/partner-reports')
+    }
+
+    steps {
+        virtualenv {
+            name('retirement-partner-reporter')
+            nature('shell')
+            command(readFileFromWorkspace('platform/resources/retirement-partner-reporter.sh'))
+        }
+    }
+
+    publishers {
+        // After all the build steps have completed, cleanup the workspace in
+        // case this worker instance is re-used for a different job.
+        wsCleanup()
```

Comment:
Can you add a TODO comment about coming back to this list of publishers and adding an email alert?  Reference the following tickets:

* feature addressed in PLAT-2138
* which is blocked by TE-2618